### ISSUE TITLE
Annoying cursor issue in Quarto Visual Editor for python code cell

### ISSUE THREAD

**Issue by vidyasagarbhargava** [quarto-dev/quarto#340]
The cursor comes back to the start of python code cell after I switch back from other screen using cmd+tab.

vscode Version: 1.85.1 

https://github.com/quarto-dev/quarto/assets/5356632/1bb84b80-96c0-455c-a076-9b5a3fabe841

**Comment by falbukrek** [comment]
The same issue occurs for R code blocks as well.

**Comment by jjallaire** [comment]
Yes, this is a known issue which we regrettably don't have a fix for yet. It's a bit harder than it appears as the visual editor is an iframe which VS Code itself is focusing directly with and then the code block yet another component that has non-trivial internal focus management code. Hopefully we will be able to sort this out soon!
